src/xnatdcat/xnat_parser.py

```diff
@@ -158,3 +158,57 @@ def xnat_to_RDF(session: XNATSession, config: Dict) -> Graph:
         logger.warning("There were %d projects with invalid data for DCAT generation", failure_counter)
 
     return export_graph
+
+
+def split_keywords(xnat_keywords: Union[str, None]) -> List[str]:
+    """Takes an XNAT keyword list and splits it up into a list of keywords
+
+    Removes periods, colons and semicolons from the keywords
+
+    Parameters
+    ----------
+    xnat_keywords : str
+        String containing all keywords (space-separated)
+
+    Returns
+    -------
+    List[str]
+        List of keywords, empty list if there are no keywords
+    """
+    keyword_list = []
+    if xnat_keywords:
+        if len(xnat_keywords.strip()) > 0:
+            keyword_list = [
+                kw.strip()
+                for kw in xnat_keywords.strip().replace(".", " ").replace(",", " ").replace(";", " ").split(" ")
```

Comment:
consider using re:
```
re.split(';|, |\.| ', string_to_split)
```
and then just filter out empty strings

Reply:
Good suggestion, taking it :)